settings.gradle

```diff
@@ -14,3 +14,5 @@ include ':api-tester'
 include 'ui:debugview'
 include ':ui:revenuecatui'
 include ':examples:paywall-tester'
+include ':test-apps:testpurchasesandroidcompatibility'
+include ':test-apps:testpurchasesuiandroidcompatibility'
```

Comment:
Note that I've included them in the main project instead of depending on the deployed build, so it's easier to use.